spec/search_builders/curation_concerns/work_search_builder_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'spec_helper'
+
+RSpec.describe CurationConcerns::WorkSearchBuilder do
```

Comment:
This is the first of three references to `CurationConcerns` in this file. Change to `Hyrax`.